README.md

```diff
@@ -1,193 +1,425 @@
-#### Get label link
+
+More information: https://myparcelnl.github.io/api/#6_F
+
+## List of classes and their methods
+This is a list of all the classes in this SDK and their available methods.
+
+### Models
+`\MyParcelNL\Sdk\src\Model`
+
+#### MyParcelConsignment
+IMPORTANT! Access these methods through `Repository\MyParcelConsignmentRepository`, which extends `MyParcelConsignment`.
+
+```\MyParcelNL\Sdk\src\Model\MyParcelConsignment.php```
 ```php
-$myParcelCollection
-    ->setLinkOfLabels()
-    ->getLinkOfLabels()
+$consignment = (new \MyParcelNL\Sdk\src\Model\Repository\MyParcelConsignmentRepository())
+    ->setApiKey('api_key_from_MyParcel_backoffice')
+    ->setReferenceId('Order 1203')
+    
+    // Recipient/address: https://myparcelnl.github.io/api/#7_B
+    ->setPerson('Piet Hier')    // Name
+    ->setEmail('test@test.nl')  // E-mail address
+    ->setPhone('+31 612345678') // Phone number
+    ->setCompany('Piet BV')     // Company
+    
+    ->setFullStreet('Plein 1945 55b')   // Street, number and suffix in one line 
+    // OR send the street data separately:
+    ->setStreet('Plein 1945')           // Street
+    ->setNumber((string)55)             // Number
+    ->setNumberSuffix('b')              // Suffix
+    
+    ->setCity('Amsterdam')      // City
+    ->setPostalCode('2231JE')   // Postal code
+    ->setCountry('NL')          // Country                
+            
+    // Available package types:
+    // 1: Package (default)
+    // 2: Mailbox package
+    // 3: Letter
+    ->setPackageType(1)
+
+    // Options (https://myparcelnl.github.io/api/#6_A_3)
+    ->setOnlyRecipient(false)   // Deliver the package only at address of the intended recipient. This option is required for Morning and Evening delivery types.
+    ->setSignature(true)        // Recipient must sign for the package. This option is required for Pickup and Pickup express delivery types. 
+    ->setReturn(true)           // Return the package to the sender when the recipient is not home.
+    ->setLargeFormat(false)     // Must be specified if the dimensions of the package are between 100x70x50 and 175x78x58 cm. 
+    ->setInsurance(250)         // Allows a shipment to be insured up to certain amount. Only packages (package type 1) can be insured. 
+    
+    ->setLabelDescription('Order 10034') // This description will appear on the shipment label for non-return shipments. 
+        
+    // Delivery: https://myparcelnl.github.io/api/#8
+    ->setDeliveryType()
+    ->setDeliveryDate()
+    ->setDeliveryRemark()    
+        
+    // Set pickup location
+    ->setPickupLocationName()
+    ->setPickupStreet()
+    ->setPickupNumber()
+    ->setPickupPostalCode()
+    ->setPickupCity()
+   
+    // Non-EU shipment attributes: see https://myparcelnl.github.io/api/#7_E
+    ->setInvoice()
+    ->setContents()
+    ->addItem()
+    
+    // Other attributes
+    ->setBarcode()
+    ->setMyParcelConsignmentId()
+    ->setShopId() 
+    ->setStatus();
+
```

Comment:
Deze public functions moet je niet noemen in de readme, en mankeren met `@internal`